src/LinuxTracing/LeafFunctionCallManagerTest.cpp

```diff
@@ -149,15 +149,16 @@ class LeafFunctionCallManagerTest : public ::testing::Test {
 
 CallchainSamplePerfEventData BuildFakeCallchainSamplePerfEventData(
     const std::vector<uint64_t>& callchain) {
-  CallchainSamplePerfEventData event_data{
-      .pid = 10,
-      .tid = 11,
-      .regs = make_unique_for_overwrite<perf_event_sample_regs_user_all>(),
-      .data = make_unique_for_overwrite<uint8_t[]>(13)};
+  CallchainSamplePerfEventData event_data{.pid = 10,
+                                          .tid = 11,
+                                          .regs = make_unique_for_overwrite<uint64_t[]>(20),
```

Comment:
```suggestion
                                          .regs = make_unique_for_overwrite<uint64_t[]>(sizeof(perf_event_sample_regs_user_all),
```

Reply:
Shouldn't this at least be `sizeof(perf_event_sample_regs_user_all) / sizeof(uint64_t)`? We need the number of elements not bytes, right?

Reply:
You are totally right.

Reply:
Should I replace 20 with `sizeof(perf_event_sample_regs_user_all) / sizeof(uint64_t)` or leave it as 20?

Reply:
Added a const in the same file where `perf_event_sample_regs_user_all` is defined and used that const everywhere instead of 20.